contracts/sol6/SanityRatesGasPrice.sol

```diff
@@ -0,0 +1,72 @@
+pragma solidity 0.6.6;
+
+import "./utils/Utils5.sol";
+import "./utils/Withdrawable3.sol";
+
+contract SanityRatesGasPrice is Withdrawable3, Utils5 {
+    struct SanityData {
+        uint224 tokenRate;
+        uint32 reasonableDiffInBps;
+    }
+
+    mapping(address => SanityData) public sanityData;
+    uint256 public maxGasPriceWei = 50 * 1000 * 1000 * 1000; // 50 gwei
+
+    event SanityMaxGasPriceSet(uint256 maxGasPrice);
+
+    constructor(address _admin) public Withdrawable3(_admin) {
```

Comment:
Withdrawable3(_admin) alr checked and set value for admin, no need to do anything here